app/code/Magento/Catalog/Setup/Patch/Data/UpdateProductDescriptionOrder.php

```diff
@@ -0,0 +1,86 @@
+<?php
```

Comment:
@omiroshnichenko @bluemwhitew do we need to add new integration tests for these new php functions?

Reply:
no